src/async_fuse/memfs/s3_metadata.rs

```diff
@@ -176,43 +167,22 @@ impl MetaData for S3MetaData {
     }
 
     #[instrument(skip(self), err, ret)]
-    async fn read_helper(&self, ino: INum) -> DatenLordResult<(u64, SystemTime)> {
-        let open_file = self.open_files.get(ino);
-
-        let (mtime, file_size) = {
-            let attr = open_file.read().attr;
-            (attr.mtime, attr.size)
-        };
-
-        // If now is after atime + 1s, update atime
-        let atime = open_file.read().attr.atime;
-        let now = std::time::SystemTime::now();
-        let result = now
-            .duration_since(atime)
-            .unwrap_or_else(|_| Duration::new(0, 0))
-            .as_secs();
-        if result > 1 {
-            open_file.write().attr.atime = now;
-            let (res, retry) = retry_txn!(TXN_RETRY_LIMIT, {
-                let mut txn = self.kv_engine.new_meta_txn().await;
-                let mut node = self.get_inode_from_txn(txn.as_mut(), ino).await?;
-                let mut attr = node.get_attr();
-                attr.atime = now;
-                node.set_attr(attr);
-                txn.set(
-                    &KeyType::INum2Node(ino),
-                    &ValueType::Node(node.to_serial_node()),
-                );
-                (txn.commit().await, ())
-            });
-            FILESYSTEM_METRICS.observe_storage_operation_throughput(retry, "read");
-            res?;
-        }
-        Ok((file_size, mtime))
+    async fn open_local(
+        &self,
+        _context: ReqContext,
+        _ino: u64,
+        _flags: u32,
+    ) -> DatenLordResult<u64> {
+        return Ok(self.allocate_fd());
     }
 
     #[instrument(skip(self), err, ret)]
-    async fn open(&self, context: ReqContext, ino: u64, flags: u32) -> DatenLordResult<u64> {
+    async fn open_remote(
+        &self,
+        context: ReqContext,
+        ino: u64,
+        flags: u32,
+    ) -> DatenLordResult<(u64, FileAttr)> {
```

Comment:
I don't think it's nessecary to name it 'remote'

Reply:
Compared to `open_local` method, this method will get remote attr from kvengine.